<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выберите пользователя</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Добро пожаловать в службу заказа еды.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="client_Mode_Trigger">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Я клиент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manager_Mode_Trigger">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Я менеджер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_Mode_Trigger">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Я администратор</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
